bookmark.html

```diff
@@ -4,19 +4,81 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
     <title>Quiz-App | Bookmarks</title>
   </head>
   <body>
     <header>
       <h1>Bookmarks</h1>
     </header>
-    <main></main>
+    <main>
+      <article>
+        <!-- <img class="bookmark" src="#" alt="bookmark" /> -->
+        <span class="material-symbols-outlined">bookmark</span>
+        <h2>Question</h2>
+        <p class="question">
+          Question paragraph Lorem ipsum, dolor sit amet consectetur adipisicing
+          elit. Tenetur a, enim perspiciatis dolor reprehenderit eius, libero
+          impedit debitis odio asperiores iusto aliquid ipsam cum facere amet
+          aliquam illum quidem et!
+        </p>
+        <button class="show-button">Button text</button>
```

Comment:
haha, you can absolutely render a descriptive name here ;) 

```suggestion
        <button class="show-button">show Answer</button>
```